src/main/java/codesquad/cafe/controller/UserController.java

```diff
@@ -0,0 +1,50 @@
+package codesquad.cafe.controller;
+
+import codesquad.cafe.domain.User;
+import codesquad.cafe.dto.UserRequestDto;
+import codesquad.cafe.service.UserService;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+
+import java.util.List;
+
+@Controller
+@RequestMapping("/users")
+public class UserController {
+
+    private final UserService userService;
+
+    public UserController(UserService userService) {
+        this.userService = userService;
+    }
+
+    @GetMapping("/join")
+    public String showJoinForm() {
+        return "user/form";
+    }
+
+    @PostMapping("")
+    public String registerUser(UserRequestDto userRequestDto) {
```

Comment:
```suggestion
    public String registerUser(final UserRequestDto userRequestDto) {
```

위 코드에 대해서 어떻게 생각하시나요?

hint. 불변성, kotlin 에서는 변수를 어떻게 다룰까?